src/rgw/rgw_reshard.cc

```diff
@@ -993,8 +989,11 @@ int RGWReshard::process_single_logshard(int logshard_num)
 	}
 
 	RGWBucketReshard br(store, bucket_info, attrs, nullptr);
-	ret = br.execute(entry.new_num_shards, max_entries, false, nullptr,
-			 nullptr, this);
+        // ret = br.execute(entry.new_num_shards, max_entries, false, nullptr,
```

Comment:
should not leave commented code

Reply:
Removed & apology, this hunk was committed by mistake,
(it was changed for testing purpose only).
I did not intend to revert commit cc27e35